[BUG] Windows tests failing due to pip permission issues

https://github.com/voxel51/fiftyone/runs/1086721641

Error 1:

```
Run pip install --upgrade pip setuptools wheel
Collecting pip
  Downloading pip-20.2.3-py2.py3-none-any.whl (1.5 MB)
Collecting setuptools
  Downloading setuptools-50.3.0-py3-none-any.whl (785 kB)
Collecting wheel
  Downloading wheel-0.35.1-py2.py3-none-any.whl (33 kB)
Installing collected packages: pip, setuptools, wheel
  Attempting uninstall: pip
    Found existing installation: pip 20.2.2
    Uninstalling pip-20.2.2:
      Successfully uninstalled pip-20.2.2
ERROR: Could not install packages due to an EnvironmentError: [WinError 5] Access is denied: 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pip-uninstall-od1eiz3s\\pip.exe'
Consider using the `--user` option or check the permissions.

##[error]Process completed with exit code 1.
```

Error 2:

```
Run find tests/isolated/ -name '*.py' -print0 | xargs -0 --verbose -n1 python tests/utils/pytest_wrapper.py --verbose
python tests/utils/pytest_wrapper.py --verbose tests/isolated/import_deps_test.py 
Traceback (most recent call last):
  File "tests/utils/pytest_wrapper.py", line 7, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
python tests/utils/pytest_wrapper.py --verbose tests/isolated/import_time_test.py 
Traceback (most recent call last):
  File "tests/utils/pytest_wrapper.py", line 7, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
python tests/utils/pytest_wrapper.py --verbose tests/isolated/server_tests.py 
Traceback (most recent call last):
  File "tests/utils/pytest_wrapper.py", line 7, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
python tests/utils/pytest_wrapper.py --verbose tests/isolated/service_tests.py 
Traceback (most recent call last):
  File "tests/utils/pytest_wrapper.py", line 7, in <module>
    import psutil
ModuleNotFoundError: No module named 'psutil'
##[error]Process completed with exit code 123.
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -66,8 +66,10 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
+        # use `python -m pip` to work around Windows permission issues when
+        # upgrading pip
         run: |
-          pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade pip setuptools wheel
       - name: Download fiftyone wheel
         uses: actions/download-artifact@v2
         with:

